### PR TITLE
Fixes #1398 Create tests for individual_post.dart

### LINKED ISSUE #1234: Views: Create tests for individual_post.dart
The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/after_auth_screens/feed/individual_post.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- These tests must be placed in a single file with the name `test/views/after_auth_screens/feed_test/individual_post_test.dart`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/views/after_auth_screens/feed/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

### LINKED ISSUE #1398: Views: Create tests for individual_post.dart
The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/after_auth_screens/feed/individual_post.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- These tests must be placed in a single file with the name `test/views/after_auth_screens/feed/individual_post_test.dart`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/views/after_auth_screens/feed/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

### PR DESCRIPTION
**Pull request by Ayush0Chaudhary**
**What kind of change does this PR introduce?**
[Fixes](https://github.com/PalisadoesFoundation/talawa/commit/f81970b4a987811f148a84be370ad04c725671d7) https://github.com/PalisadoesFoundation/talawa/issues/1398 [Create tests for individual_post.dart](https://github.com/PalisadoesFoundation/talawa/commit/f81970b4a987811f148a84be370ad04c725671d7)

**Issue Number:**

Fixes #1398
and Fixes #1234 as it was a duplicate issue

**Did you add tests for your changes?**
YES

**Does this PR introduce a breaking change?**
NO

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
YES

### CODE REVIEW

**Comment by Ayush0Chaudhary** [comment]
@palisadoes @noman2002 please review

**Comment by palisadoes** [comment]
@TheHazeEffect Please review

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,10 +9,12 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
    <application
-        android:label="talawa"
+       android:label="talawa"
         android:icon="@mipmap/ic_launcher"
         android:allowBackup="false"
        tools:replace="android:label">
+
+       android:enableOnBackInvokedCallback="true"
```
Comment (noman2002) [inline android/app/src/main/AndroidManifest.xml:17]: Why we need this ?

Comment (Ayush0Chaudhary) [reply android/app/src/main/AndroidManifest.xml]: removed it. Was trying something

Comment (Ayush0Chaudhary) [reply android/app/src/main/AndroidManifest.xml]: @noman2002 @TheHazeEffect

**Comment by Ayush0Chaudhary** [comment]
@noman2002 @palisadoes there are 2 issue for same test #1398 and #1234

**Review by noman2002 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**